angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.ts

```diff
@@ -41,6 +41,7 @@ export class CustomAppComponent {
     public selectedRadio = 'mango';
     public activeTabId = 'tab-1';
     public activeAnchorTabId = 'a-tab-2';
+    public markdownString = 'Supported rich text formatting options:\n1. **Bold**\n2. *Italics*\n3. Numbered lists\n   1. Option 1\n   2. Option 2\n4. Bulleted lists\n   * Option 1\n   * Option 2\n5. Direct link: <https://nimble.ni.dev/>';
```

Comment:
You can use template string literals to make this look nicer andd easier to edit as markdown:

```suggestion
    public markdownString = `Supported rich text formatting options:
1. **Bold**
2. *Italics*
3. Numbered lists
   1. Option 1
   2. Option 2
4. Bulleted lists
   * Option 1
   * Option 2
5. Direct link: <https://nimble.ni.dev/>
`;
```

Reply:
Changed to template string literals